tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
+    "lib": ["es2015", "dom"],
```

Comment:
Can we add some kind of test just to make sure we don't entirely break things?

Reply:
I will!